region/wreckedship/main.json

```diff
@@ -1552,6 +1673,17 @@
               "devNote":  "This is unlocked and yielded from killing the enemies, which are free to kill"
             }
           ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Covern, Atomic, or Kihunter",
+                  "notable": false,
+                  "requires": []
```

Comment:
bottom doors use ice afaik
maybe you meant right door

Reply:
I did indeed. Thanks, I believe those are both fixed now